### DISCUSSION TITLE
New upcoming release - eCAL 5.8

### DISCUSSION THREAD

**Discussion by rex-schilasky** [eclipse-ecal/ecal#144] [General]
A new release is in preparation that will complete the eCAL application family (eCALMon, eCALRec and eCALPlay) with a new outstanding application **eCALSys**.

Details will follow soon ...

**Comment by rex-schilasky** [comment]
It's done. eCAL 5.8.0 is out now. Welcome eCALSys as new member of the eCAL application family. Just try it out, start any combination of eCAL nodes on a single machine or distributed in a network and then use eCALSys to create a configuration snapshot out of it that can be used to reproduce, modify this setup later on again and again. Magic :-)

**Comment by pratikmankawde** [comment]
Hi,
I am trying eCal5 for a personal project. I can see the docs mentioning about the high performance and speed. But when I tested it, it seems to not perform as expected in terms of message capture at subscriber. The use case I am trying it very simple. I am publishing messages continuously on a topic and the subscriber is only receiving some of them. Is there no queue at the subscriber's end? Why are the messages getting lost? In the samples you have provided, you use sleep in the publisher thread. Is that compulsory? It hinders performance. BTW I am using protocol buffers for data. Any suggestion would be helpful.

**Reply by FlorianReimold** [reply]
eCAL traffic does not have any kind of "back channel" and thus no guaranteed quality of service. It is not like a TCP Stream, that will only send as fast as the receiver can receive. With eCAL you can publish as fast as your hardware lets you. And if you publish faster than the reciever can receive, you will loose messages.

Can you please create a new discussion topic, where we can take a deeper look into your setup? A comment to this Version announcement isn't the proper place for that.

**Reply by pratikmankawde** [reply]
Yeah, sorry to add a comment here instead of starting a new discussion, but I don't seem to have permissions to start a new discussion. I don't see any button for that. I can however add to the issues page,
